rs/nns/governance/canister/canister.rs

```diff
@@ -167,13 +167,17 @@ fn schedule_timers() {
     schedule_adjust_neurons_storage(Duration::from_nanos(0), NeuronIdProto { id: 0 });
     schedule_prune_following(Duration::from_secs(0));
     schedule_spawn_neurons();
+
+    // TODO(NNS1-3446): Delete. (This only needs to be run once, but can safely be run multiple times).
+    schedule_backfill_voting_power_refreshed_timestamps(Duration::from_secs(0));
 }
 
 // Seeding interval seeks to find a balance between the need for rng secrecy, and
 // avoiding the overhead of frequent reseeding.
 const SEEDING_INTERVAL: Duration = Duration::from_secs(3600);
 const RETRY_SEEDING_INTERVAL: Duration = Duration::from_secs(30);
 const PRUNE_FOLLOWING_INTERVAL: Duration = Duration::from_secs(10);
+const BACKFILL_VOTING_POWER_REFRESHED_TIMESTAMPS_INTERVAL: Duration = Duration::from_secs(10);
```

Comment:
Nit.

Since this is a one-off, maybe we could save a bit of subnet resources here?

```suggestion
const BACKFILL_VOTING_POWER_REFRESHED_TIMESTAMPS_INTERVAL: Duration = Duration::from_secs(300);
```